lib/matplotlib/tests/test_axes.py

```diff
@@ -1749,11 +1749,12 @@ def test_bar_hatches(fig_test, fig_ref):
 
 def test_pandas_minimal_plot(pd):
     # smoke test that series and index objcets do not warn
-    x = pd.Series([1, 2], dtype="float64")
-    plt.plot(x, x)
-    plt.plot(x.index, x)
-    plt.plot(x)
-    plt.plot(x.index)
+    for x in [pd.Series([1, 2], dtype="float64"),
+              pd.Series([1, 2], dtype="Float32")]:
```

Comment:
Let's just put in
```suggestion
              pd.Series([1, 2], dtype="Float64")]:
```
For clarity and merge. This not code run by the user. The worst thing that can happen is that the test starts failing if numpy changes something.